tutor/resources/styles/variables/book-content.less

```diff
@@ -60,6 +61,11 @@
 @openstax-book-concepts_biology-bg: @openstax-book-orange;
 @openstax-book-concepts_biology-accent: @openstax-book-orange-accent;
 
+// History
+@openstax-book-ap_us_history-fg: rgb(241, 118, 72);
+@openstax-book-ap_us_history-bg: rgb(35,  48,  99);
```

Comment:
hmmm...these are awfully close `@openstax-tertiary` aka `@dark-blue` and `@openstax-primary` aka `@orange`  https://github.com/openstax/tutor-js/blob/master/shared/resources/styles/variables/colors.less#L16